skimage/transform/_hough_transform.pyx

```diff
@@ -331,6 +331,10 @@ def _probabilistic_hough_line(cnp.ndarray img, int threshold,
         Increase the parameter to merge broken lines more aggresively.
     theta : 1D ndarray, dtype=double
         Angles at which to compute the transform, in radians.
+    use_seed : bool
```

Comment:
We can just use the conventional Python syntax: `seed=None`, and do nothing unless the seed is specified, i.e.:

```
if seed is not None:
    ...
```